source/index.html.md.erb

```diff
@@ -13832,7 +13876,7 @@ Gets a list of user-created Templates that the user has access to.
 
 # Token
 
-The Smartsheet API utilizes OAuth 2.0 for Authentication and Authorization. An **Authorization** HTTP header containing an Access Token is required to authenticate requests.  For more information, see [Authentication](#authentication-and-access-tokens).
+The Smartsheet API utilizes OAuth 2.0 for Authentication and Authorization. An **Authorization** HTTPS header containing an Access Token is required to authenticate requests.  For more information, see [Authentication](#authentication-and-access-tokens).
```

Comment:
AFAICT, the convention is to call these HTTP headers, even if the actual transport is HTTPS

Reply:
Done. I only found two places.